robottelo/helpers.py

```diff
@@ -359,23 +359,22 @@ def get_available_capsule_port(port_pool=None):
                 )
             )
     # returns a list of strings
-    fuser_cmd = ssh.command(
-        'fuser -n tcp {{{0}..{1}}} 2>&1 | awk -F/ \'{{print$1}}\''.format(
-            port_pool[0], port_pool[-1]
-        )
+    ss_cmd = ssh.command(
+        f"ss -tnaH sport ge {port_pool[0]} sport le {port_pool[-1]}"
+        " | awk '{print $4}' | awk -F ':' '{print $NF}' | sort | uniq"
```

Comment:
Every time I see command output piped to same command, I wonder if there is a way to do everything on single command invocation. Usually there is.
```
" | awk '{n=split($4, p, ":"); print p[n]}' | ...
```

Also, if you don't pass any flag to `uniq`, you can tell `sort` to skip duplicates:
```
| sort -u
```
That flag has been there since at least 1980s, if compatibility would be your concern.

Reply:
Updated the command to use `awk` only once, and to use `sort -u` instead of `sort | uniq`. Thanks!

Re-ran tests, and the test output is the same as above.